README.md

```diff
@@ -33,11 +33,37 @@ npm install --save systemjs-webpack-interop
 yarn add systemjs-webpack-interop
 ```
 
-## Public Path
+## Setting Public Path
 
 systemjs-webpack-interop will [dynamically set the webpack public path](https://webpack.js.org/guides/public-path/#on-the-fly) based on the URL that a SystemJS module was downloaded from.
 
-### Newer versions of webpack
+### As a Webpack Plugin
+
+You can set the public path by adding the SystemJSPublicPathWebpackPlugin.
+
+```js
+// webpack.config.js
+const SystemJSPublicPathWebpackPlugin = require("systemjs-webpack-interop/SystemJSPublicPathWebpackPlugin");
+
+module.exports = {
+  plugins: [
+    new SystemJSPublicPathWebpackPlugin({
```

Comment:
My thought is we can add this to webpack-config-single-spa, and then remove the set-public-path.js files entirely from all the templates.